## /bin/sh -e simulate dpatch for prepend_chlog.py
## Feature
## daemon_color.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
## DP: Enable color support even in a daemonized Asterisk. This is
## DP: done by assuming the default value 'xterm' for TERM.
## DP: See upstream bug http://bugs.digium.com/view.php?id=9048

Index: asterisk-1.2.27/term.c
===================================================================
--- asterisk-1.2.27.orig/term.c	2008-03-19 11:32:20.000000000 +0100
+++ asterisk-1.2.27/term.c	2008-03-19 11:33:44.000000000 +0100
@@ -79,8 +79,8 @@
 	int termfd = -1, parseokay = 0, i;
 
 	if (!term)
-		return 0;
-	if (!option_console || option_nocolor || !option_nofork)
+		term="xterm";
+	if (option_nocolor)
 		return 0;
 
 	for (i=0 ;; i++) {
